partner_document/wizard/res_partner_expired_document_wizard.py

```diff
@@ -0,0 +1,42 @@
+from datetime import date
+
+from odoo import fields, models
+
+
+class ResPartnerExpiredDocumentWizard(models.TransientModel):
+    _name = "res.partner.expired.document.wizard"
+    _description = "Partner expired document wizard"
+
+    document_type_ids = fields.Many2many(
+        comodel_name="partner.document.type",
+        required=True,
+        ondelete="restrict",
```

Comment:
"ondelete" only applies on m2o fields, please remove it.
Add relation="partner_document_expired", column1="expired_document_wizard_id", column2="document_type_id"